ir/src/main/scala/org/scalajs/core/ir/Utils.scala

```diff
@@ -66,6 +66,35 @@ object Utils {
     // scalastyle:on return
   }
 
+  def printEscapeJS(str: String, out: java.io.Writer): Unit = {
```

Comment:
No chance to factor out the common parts with `createJSString`? :( If not, this needs 2 `!!!` code duplicate comments.